src/index.html

```diff
@@ -4,10 +4,51 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./../src/styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+
+    <div class="product-card" data-qa="card">
+
+      <div class="photo"></div>
+
+      <section class="description">
+        <h1 class="description__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        <h3 class="description__product-code">Product code: 195434</h3>
+      </section>
+
+      <div class="rating">
+
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+
+        <h3 class="rating__reviews">Reviews: 5</h3>
+      </div>
+
+      <div class="price">
+        <h3 class="price__text">Price:</h3>
+        <h1 class="price__numbers">&#36;2,199</h1>
```

Comment:
This tag use for title of the page, so you need to use another tag here
```suggestion
        <p class="price__numbers">&#36;2,199</p>
```